chall02/jtrancos.py

```diff
@@ -0,0 +1,38 @@
+# **************************************************************************** #
+#                                                                              #
+#                                                         :::      ::::::::    #
+#    jtrancos.py                                        :+:      :+:    :+:    #
+#                                                     +:+ +:+         +:+      #
+#    By: jtrancos <marvin@42.fr>                    +#+  +:+       +#+         #
+#                                                 +#+#+#+#+#+   +#+            #
+#    Created: 2020/04/06 19:47:21 by jtrancos          #+#    #+#              #
+#    Updated: 2020/04/06 22:58:06 by jtrancos         ###   ########.fr        #
+#                                                                              #
+# **************************************************************************** #
+
+#!/usr/bin/env python
```

Comment:
Good 👍 but you should read documentation about shebang, the Python version is missing you should write `python3`, not just `python`